src/utils/ScorecardRules.ts

```diff
@@ -36,6 +51,14 @@ export const isNotEvaluatedRuleOutcome = (
 ): rule is NotEvaluatedRuleOutcome =>
   rule?.type === RuleOutcomeType.NOT_EVALUATED;
 
+export const getApplicableRules = (
+  rules: RuleOutcome[],
+): ApplicableRuleOutcome[] => {
+  return rules.filter(rule =>
+    isApplicableRuleOutcome(rule),
+  ) as ApplicableRuleOutcome[];
```

Comment:
nit: can use shorthand and probably don't need to cast. Feel free to ignore if TSC is being silly
```suggestion
  return rules.filter(isApplicableRuleOutcome);
```